### PR TITLE
Replacing proxy and control servers on staging environment

### PR DESCRIPTION
**Pull request by sanjay2916**
##### ENVIRONMENTS AFFECTED
Staging

### CODE REVIEW

File: environments/staging/terraform.yml [hunk 1]
```diff
@@ -216,17 +216,17 @@ servers:
 
 
 proxy_servers:
-  - server_name: "proxy1-staging"
+  - server_name: "proxy2-staging"
```
Comment (kaapstorm) [inline environments/staging/terraform.yml:219]: `proxy3-staging`?

Comment (kaapstorm) [reply environments/staging/terraform.yml:219]: Sorry, nvm, I just noticed the "-" below -- it's a separate server, not a repeated key.

File: environments/staging/aws-resources.yml [hunk 2]
```diff
@@ -35,8 +35,8 @@ pgproxy2-staging.instance_id: i-0638ff6218ed9f753
 pillow3-staging: 10.201.10.68
 pillow3-staging.instance_id: i-0ef555fec7fe64804
 proxy1-staging: 10.201.20.208
```
Comment (kaapstorm) [inline environments/staging/aws-resources.yml:37]: Should `proxy1-staging` still be here?

File: environments/staging/aws-resources.yml [hunk 3]
```diff
@@ -35,8 +35,8 @@ pgproxy2-staging.instance_id: i-0638ff6218ed9f753
...
-proxy1-staging.instance_id: i-02e1fe640296f8db9
-proxy1-staging.public_ip: 3.89.61.176
+proxy3-staging: 10.201.20.47
+proxy3-staging.instance_id: i-0a53d5199910fc2b7
```
Comment (kaapstorm) [inline environments/staging/aws-resources.yml:39]: Do we need to include `proxy3-staging.public_ip`?

File: environments/staging/aws-resources.yml (hunk 2 above)
Comment (sanjay2916) [reply environments/staging/aws-resources.yml]: I have removed this

File: environments/staging/aws-resources.yml (hunk 3 above)
Comment (sanjay2916) [reply environments/staging/aws-resources.yml]: No , I have re setup in private subnet

**Review by kaapstorm — APPROVED** [review]

**Pull request merged**